src/containers/sidebar/index.js

```diff
@@ -105,7 +110,7 @@ class ChatBar extends React.Component{
             <MuiThemeProvider>
                 <SelectableList defaultValue={0} changeChat={this._handleChatChange}>
                     <ListItem disabled={true}>
-                        <SearchBar hintText="Search Chats" />
+                        <TextField className="searchbar" hintText="Search Chats" value={this.setState.search} onChange={this._handleSearchChange.bind(this)} />

```

Comment:
this.setState.search?

Reply:
after searching, the chat-list has to be re-rendered again i used setState to ensure that something was typed in the searchbar then render the sorted chats list. Should I change it?

Reply:
should be this.state.search

Reply:
I agree. I think this.setState.search will be undefined.